clients/src/main/java/org/apache/kafka/clients/consumer/internals/CompletedFetch.java

```diff
@@ -135,7 +135,8 @@ void recordAggregatedMetrics(int bytes, int records) {
     /**
      * Draining a {@link CompletedFetch} will signal that the data has been consumed and the underlying resources
      * are closed. This is somewhat analogous to {@link Closeable#close() closing}, though no error will result if a
-     * caller invokes {@link #fetchRecords(FetchConfig, int)}; an empty {@link List list} will be returned instead.
+     * caller invokes {@link #fetchRecords(FetchConfig, Deserializers, int)}; an empty {@link List list} will be
```

Comment:
I think I wrote the initial comment 😬

I was trying to explain that `drain()`-ing a `CompletedFetch` is like `close()`-ing it, in that we free its resources. But it's _unlike_ `close()` because we can technically still call `fetchRecords()` on it without throw some sort of `IllegalStateException` or anything.

> It is OK for the caller to invoke `fetchRecords()` on a `CompletedFetch` that has been `drain()`-ed. An exception won't be thrown; it will simply return no records.

Or I can just scrap the whole comment altogether?